group_project_v2/project_api/api_implementation.py

```diff
@@ -391,3 +396,23 @@ def get_member_data(self, user_id):
         if user_organizations:
             user_details.organization = user_organizations[0]['display_name']  # and a string here
         return user_details
+
+    @memoize_with_expiration()
+    def get_organization_by_id(self, org_id):
+        """
+        :param org_id:
+        :return: Requested organization
+        :rtype: OrganisationDetails
+        """
+        return OrganisationDetails(**self.send_request(GET, (ORGANIZATIONS_API, org_id)))
+
+    @memoize_with_expiration()
+    def get_user_groups(self, user_id):
```

Comment:
@jbzdak nit: might make sense to add group type, similar to [`get_groups_of_type`](https://github.com/mckinseyacademy/mcka_apros/blob/master/lib/authorization.py#L17)